Assets/Scripts/Interactables/InteractableObject.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(typeof(Outline), typeof(SceneSwitcher))]
+public class InteractableObject : MonoBehaviour, IInteractable
```

Comment:
Could be renamed to "InteractableSceneLoader" or something so that we don't mix it with other interactables

Reply:
This one was kinda a shortcut cuz the idea was that interacting with the object would call a sort of task complete function from the work/self care tasks. We don't have a task system yet so I made the InteractableObject switch the scene directly.